src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/SecurityHelper.cs

```diff
@@ -836,15 +796,12 @@ internal static PermissionSet EnvelopePermissionSet
         private static PermissionSet CreateEnvelopePermissionSet()
         {
             PermissionSet permissionSet = new PermissionSet(PermissionState.None);
```

Comment:
Simplify as `return new PermissionSet(PermissionState.None);`

Note that this is only called in one place - so you can directly change `SecurityHelper.EnvelopePermissionSet.get` to incorporate this, and delete `CreateEnvelopePermissionSet()`.